roles/mongodb/tasks/main.yml

```diff
@@ -31,18 +31,18 @@
 - debug:
     var: is_raspbian
 
-- name: EXIT 'mongodb' ROLE & CONTINUE, IF 'is_debian and not is_raspbian' i.e. TRUE DEBIAN (where MongoDB no longer exists)
-  fail:    # FORCE IT RED THIS ONCE!
-    msg: ATTEMPTED MongoDB INSTALLATION WITH (TRUE) DEBIAN, which is no longer supported.  Nevertheless IIAB will continue (consider this a warning!)
-  when: is_debian and not is_raspbian
-  #when: (is_debian and not is_raspbian) and (not is_debian_8) and (not is_debian_9)    # Test for Debian 10+
-  ignore_errors: yes
+#- name: EXIT 'mongodb' ROLE & CONTINUE, IF 'is_debian and not is_raspbian' i.e. TRUE DEBIAN (where MongoDB no longer exists)
+#  fail:    # FORCE IT RED THIS ONCE!
+#    msg: ATTEMPTED MongoDB INSTALLATION WITH (TRUE) DEBIAN, which is no longer supported.  Nevertheless IIAB will continue (consider this a warning!)
+#  when: is_debian_10 and (ansible_architecture == "aarch64")
+#  #when: (is_debian and not is_raspbian) and (not is_debian_8) and (not is_debian_9)    # Test for Debian 10+
+#  ignore_errors: yes
 
 # ELSE...
 
 - name: Install/Enable/Disable/Record MongoDB (main2.yml) if is_raspbian or not is_debian, i.e. not True Debian
   include_tasks: main2.yml
-  when: is_raspbian or not is_debian
+#  when: is_raspbian or not is_debian
```

Comment:
Many of the `- name:` descriptions in mongodb/tasks/install.yml (and just above here in mongodb/tasks/main.yml) have residual mentions of `raspbian` conditions...that should be removed now presumably?

Reply:
Should main.yml and main2.yml be combined again as mongodb now can be installed on all platforms?

Reply:
Seems reasonable.  If we're pretty sure MongoDB can be installed easily on most all platforms incl vanilla Debian 10?